apps/linode-marketplace-plesk/README.md

```diff
@@ -0,0 +1,97 @@
+# Linode Plesk Deployment One-Click APP
+
+Plesk is the leading secure WordPress and website management platform. This Ansible playbook installs the latest publicly available Plesk, activates a trial license, installs essential extensions, and sets up and configures the firewall. Please allow the script around 15 minutes to finish.
+
+## Software Included
+
+| Software       | Version   | Description   |
+| :---           | :----     | :---          |
+| Dovecot        | 2.3       | An open-source IMAP and POP3 email server for Linux/UNIX-like systems, known for its performance and security. |
+| nginx          | 1.24      | A high-performance web server, reverse proxy, and HTTP cache, widely used for its efficiency, stability, and low resource consumption. |
+| Node.js        | Latest    | An open-source, cross-platform JavaScript runtime environment that executes JavaScript code outside a web browser. |
```

Comment:
```suggestion
```
This is not part of the system

Reply:
Removed. I think I may have just pulled that from https://docs.plesk.com/release-notes/obsidian/software-requirements/#s4 without double checking whether it was installed.